src/App.tsx

```diff
@@ -68,11 +66,11 @@ export const App: React.FC = () => {
         ));
       })
       .catch(() => setErrorMessage('Unable to delete a todo'))
-      .finally(() => setLoading([]));
+      .finally(() => setIsProcessed([]));
   };
 
   const clearCompletedTodos = () => {
-    todos.filter(todo => {
+    todos.map(todo => {
```

Comment:
just FYI - the map would be the correct choice if you save the result (array of Promises) in the variable and put it into Promise.all. In this case, still `forEach` would be preferred